rbc/request_internal_test.go

```diff
@@ -0,0 +1,49 @@
+package rbc
+import (
+	"reflect"
+	"testing"
+)
+
+func TestValReqRepository_Find(t *testing.T) {
+	valrepo,_:= NewValReqRepository()
+	valstruct:=ValRequest{nil,nil,nil}
+	valrepo.Save("ID1",&valstruct)
+	if valrepo.Find("ID1")!=&valstruct{
+		t.Fatalf("request %v is not found.",valrepo.recv["ID1"])
+	}
+	if len(valrepo.FindAll())!=1{
```

Comment:
I think you don't need this part at `TestValReqRepository_Find`. It's duplicate test with `TestValReqRepository_FindAll`

Reply:
I think it will be better if I check the length.